docs/units/ref_api.rst

```diff
@@ -26,6 +27,7 @@ Reference/API
 .. automodapi:: astropy.units.physical
 
 .. automodapi:: astropy.units.equivalencies
+   :noindex:
```

Comment:
I'd actually mostly want to be sure the `equivalencies` document doesn't change - we need this nice summary of all possible ones.

Reply:
Agreed!